ECOSYSTEM.md

```diff
@@ -0,0 +1,57 @@
+# The React Native Ecosystem
+
+We aim to build a vibrant and inclusive ecosystem of partners, core contributors, and community that goes beyond the main React Native GitHub repository. This document/blog post explains the roles and responsibilities of various stakeholders and provides guidelines for the community organization. The structure outlined in this document has been in place for a while but hadn't been written down before.
+
+There are three types of stakeholders:
+
+* **Partners:** Companies that are significantly invested in React Native and have been for years.
+* **Core Contributors:** Individual people who contribute to the React Native project.
+* **Community Contributors:** Individuals who support projects in the [react-native-community](https://github.com/react-native-community) organization.
+
+## Partners
+
+Partners are companies that are significantly invested in React Native and have been for years. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Facebook's partners think of React Native as a product: they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications.
+
+Our current set of partners include Callstack, Expo, Infinite Red, Microsoft and Software Mansion. Besides many engineers from these companies being core contributors, their responsibilities include:
+
+* **[Callstack](https://callstack.com/):** Manages releases, maintains the [React Native CLI](https://github.com/react-native-community/react-native-cli) and organizes [React Native EU](https://react-native.eu/)
+* **[Expo](https://expo.io/):** Builds [expo](https://github.com/expo/expo) on top of React Native to simplify app development
+* **[Infinite Red](https://infinite.red/):** Maintains the [ignite cli/boilerplate](https://github.com/infinitered/ignite), organizes [Chain React Conf](https://infinite.red/ChainReactConf)
+* **[Microsoft](https://www.microsoft.com/en-gb/):** Develops [React Native Windows](https://github.com/Microsoft/react-native-windows) for the UWP platform
+* **[Software Mansion](https://swmansion.com/):** Maintain core RN infrastructure, like JSC, Animated and popular third-party plugins.
```

Comment:
```suggestion
* **[Software Mansion](https://swmansion.com/):** Maintain core infrastructure including JSC, Animated, and other popular third-party plugins.
```

- switch "like" to "including"
- drop "RN" as it's implied, otherwise would recommend using "React Native"